### ISSUE TITLE
init fails on windows

### ISSUE THREAD

**Issue by ghost** [mimiMonads/create-vixeny#5]
OS: Windows 11 Pro for Workstations 23H2
got bun and deno from winget (bun version is 1.1.29, deno version is 2.0.0)
trying to create with `bun create vixeny`
at `Where to create the project? Enter directory name or ".":` if i try to specify any folder without full path it gives the following error

```
Project initialized successfully.
Failed to initialize the project: Error: Command failed: null init -y
'null' is not recognized as an internal or external command,
operable program or batch file.
```

however, if i do try to specify a full path it instead gives the following error:

```
150 |       projectPath = process.cwd();
151 |     }
152 |
153 |     // Create project directory if it doesn't exist
154 |     if (!fs.existsSync(projectPath)) {
155 |       fs.mkdirSync(projectPath);
               ^
EINVAL: Invalid argument
   errno: -22
 syscall: "mkdir"

      at C:\Users\Admin\AppData\Local\Temp\bunx-1534026329-create-vixeny@latest\node_modules\create-vixeny\bin\create-vixeny.mjs:155:10

Bun v1.1.29 (Windows x64)
```

<details>
<summary>small addendum</summary>
this *might* be the fault of those particular packages, will try to recreate with bun & deno downloaded from their official webpages
</details>

**Comment by mimiMonads** [comment]
this should be fixed in #4 and ready for the next release

**Comment by ghost** [comment]
it works now
